tox.ini

```diff
@@ -56,6 +56,7 @@ allowlist_externals =
     make
     rm
 deps =
+    setuptools
```

Comment:
I think we have defined setuptools in default [testenv] and no need to define in others